### ISSUE TITLE
Two semicolon in property crash minify

### ISSUE THREAD

**Issue by zoobestik** [css/csso#378]
Two semicolon in css file breaks minify. For ex.:
```css
.c .d { color: #ffffff;; }
```
Throw the error:
```
Cannot read property '0' of undefined TypeError: Cannot read property '0' of undefined
    at getPropertyDescriptor (.../css-tree/lib/utils/names.js:65:24)
    at getPropertyFingerprint (.../csso/lib/restructure/6-restructBlock.js:68:20)
    at List.<anonymous> (.../csso/lib/restructure/6-restructBlock.js:213:27)
    at List.eachRight (.../css-tree/lib/utils/list.js:182:12)
    at Object.processRule (.../csso/lib/restructure/6-restructBlock.js:211:18)
```

<img width="880" alt="screen shot 2018-04-17 at 12 26 40" src="https://user-images.githubusercontent.com/242514/38861219-ae0a5d7e-423a-11e8-8fdf-c6d54a52a5e8.png">

<img width="1171" alt="screen shot 2018-04-17 at 12 29 46" src="https://user-images.githubusercontent.com/242514/38861367-0f2c5602-423b-11e8-9a3b-5929066c668c.png">

**Comment by Avol-V** [comment]
I have the same issue:
```
TypeError: Cannot read property '0' of undefined
    at getPropertyDescriptor (node_modules/css-tree/lib/utils/names.js:65:24)
    at getPropertyFingerprint (node_modules/csso/lib/restructure/6-restructBlock.js:68:20)
    at List.<anonymous> (node_modules/csso/lib/restructure/6-restructBlock.js:213:27)
```

In *6-restructBlock.js* `declaration` is:
```js
{
	fingerprint:null,
	id:211,
	length:1,
	loc:null,
	type:"Raw",
	value:";",
}
```
So `declaration.property` is `undefined`.

**Comment by lahmatiy** [comment]
Fixed in [4.0.2](https://github.com/css/csso/releases/tag/v4.0.2)
